dev-tools/xtask/src/main.rs

```diff
@@ -40,6 +42,12 @@ struct Args {
 
 #[derive(Subcommand)]
 enum Cmds {
+    /// Generate a tarball with omicron packaged for deployment onto a4x2
+    A4x2Package(a4x2_package::A4x2PackageArgs),
+
+    /// Run a4x2 and deploy omicron onto it, and optionally run live-tests and end to end tests
+    A4x2Deploy(a4x2_deploy::A4x2DeployArgs),
+
```

Comment:
It's worth gating these on illumos and printing a nice error on other platforms.